server/index.js

```diff
@@ -90,16 +100,26 @@ const addSession = async (app) => {
     }
   });
 };
-
+const addLocalization = (app) => {
+  i18next
+    .init({
+      lng: 'ru',
+      fallbackLng: 'en',
+      debug: isDevelopment,
+      resources: { ru },
+    });
+  app.decorate('t', (key) => i18next.t(key));
+};
 export default async () => {
   const app = fastify({
     logger: {
-      prettyPrint: true,
-      level: 'error',
+      prettyPrint: isDevelopment,
+      level: 'trace',
```

Comment:
trace level is very verbose and resource consuming. it's recommended to use it in development only. For production "info" would be fine